IPs/RSKIP213.md

```diff
@@ -0,0 +1,77 @@
+# Simple Transaction Versioning System
+
+
+|RSKIP          | 213 |
+| :------------ |:-------------|
+|**Title**      |Simple Transaction Versioning System|
+|**Created**    |FEB-2021 |
+|**Author**     |SDL|
+|**Purpose**    |Sca |
+|**Layer**      |Core |
+|**Complexity** |1|
+|**Status**     |Draft |
+
+
+# **Abstract**
+
+This RSKIP proposes the addition of a version field so that transaction parsing can start after transaction version has been verified. This reduces the amount of changes required for the node and software wallets.
```

Comment:
This RSKIP proposes the addition of a version field so that transaction parsing can start after the transaction version has been verified. This reduces the amount of changes required for the node and software wallets.

Reply:
Can you indicate the exact change required instead of repeating the paragraph?

Reply:
Add the word "the" in to the sentence "This RSKIP proposes the addition of a version field so that transaction parsing can start after **the** transaction version has been verified".